packages/thirdweb/src/extensions/marketplace/direct-listings/write/createListing.ts

```diff
@@ -53,7 +53,7 @@ export type CreateListingParams = {
       /**
        * The price per token (in Ether)
        */
-      pricePerToken: string;
+      pricePerToken: string | number;
```

Comment:
if we want to do this we need to do it for everywhere we accept "ether" values, can't do it one-off just here